gateway/cli/src/main.rs

```diff
@@ -15,6 +15,9 @@ struct Cli {
     url: String,
     #[command(subcommand)]
     command: Commands,
+    /// WARNING: Passing in a password from the command line may be less secure!
+    #[clap(long)]
+    rpcpassword: String,
```

Comment:
Can we make this optional and query for the password otherwise? Much like with @jkitman's config encryption PR:

https://github.com/fedimint/fedimint/blob/a8b1abde98941d22c9c193e4c114c559f6070d47/fedimintd/src/encrypt.rs#L55-L61